mappings/net/minecraft/client/particle/ParticleHitDamage.mapping

```diff
@@ -1,5 +1,5 @@
-CLASS none/bmt
-	CLASS none/bmt$a
+CLASS none/bmt net/minecraft/client/particle/ParticleHitDamage
+	CLASS none/bmt$a Factory
```

Comment:
TBH I think just using `Factory` for inner factory classes is fine. The context is given by the wrapper class. Usage becomes `ParticleHitDamage.Factory` rather than redundant `ParticleHitDamage.FactoryEmptyHeart`.